### ISSUE TITLE
Release v1.67.3

### ISSUE THREAD

**Issue by deckhouse-BOaTswain** [deckhouse/deckhouse#11226]
:robot: A dedicated issue to run tests and deploy release [v1.67.3](https://github.com/deckhouse/deckhouse/milestone/370).

---

<details>
<summary>Release issue commands and options</summary>
<br />

You can trigger release related actions by commenting on this issue:

- `/deploy/<channel>[/<editions>]` will publish built images into the release channel.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
  - `editions` is a comma-separated list of editions `ce | ee | fe | be | se | se-plus`
- `/suspend/<channel>` will suspend released version.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
- `/e2e/run/<provider> git_ref` will run e2e using provider and an `install` image built from git_ref.
  - `provider` is one of `aws | azure | eks | gcp | openstack | static | vcd | vsphere | yandex-cloud`
  - `git_ref` is a tag `v1.67.3 | test-v1.67.3*` or a branch `main | release-1.67`
- `/e2e/run/<provider> git_ref_1 git_ref_2` will run e2e using provider and git_ref_1 and then switch to git_ref_2.
  - `provider` is one of `aws | azure | eks | gcp | openstack | static | vcd | vsphere | yandex-cloud`
  - `git_ref_1` is a release-* or main branch
  - `git_ref_2` is a release-* or main branch
- `/e2e/use/k8s/<version>` specifies which Kubernetes version to use for e2e test.
  - `version` is one of `1.26 | 1.27 | 1.28 | 1.29 | 1.30 | 1.31 | Automatic`
- `/e2e/use/edition/<edition>` specifies which edition to use for e2e test.
  - `edition` is one of `ce | ee | fe | be | se | se-plus`
- `/build git_ref` will run build for release related refs.
  - `git_ref` is a tag `v1.67.3 | test-v1.67.3*` or a branch `main | release-1.67`
  - `/build/<edition> git_ref` will also build specified edition

**Note 1:**
A single command `/e2e/run/<provider>` will run e2e with default CRI 'containerd' and Kubernetes version '1.27'.
Put `/e2e/use` options below `/e2e/run` command to set specific CRI and Kubernetes version. E.g.:

```
/e2e/run/aws main
/e2e/use/k8s/1.27
/e2e/use/k8s/1.29

This comment will run 2 e2e jobs on AWS with containerd
and with Kubernetes version 1.20 and 1.23 using image built from main branch.
```

```
/e2e/run/aws release-1.35 release-1.36
/e2e/use/k8s/1.27

This comment will create cluster in AWS using Deckhouse built from release-1.35 branch
and then switch to images built from release-1.36 branch.
'use' options are supported, cluster will use containerd and Kubernetes version 1.23.
```

**Note 2:**
'deploy', 'suspend' and 'e2e' commands should run after 'build FE' job is finished.

**Note 3:**
No autobuild for release branch. Run this command after cherry-picking into release branch:
```
/build release-1.67
```

</details>

**Comment by deckhouse-BOaTswain** [comment]
New commit [cd4504](https://github.com/deckhouse/deckhouse/commit/cd45048b8a5bba9474f0d49d48370ec08db89f2d) in branch **main**:
```
fix: check spec object in validations (#11216)

Signed-off-by: Evsyukov Denis <denis.evsyukov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [c2ab5b](https://github.com/deckhouse/deckhouse/commit/c2ab5b30e19d7ee44ca9fa5fe6f9ccba0135b1b2) in branch **main**:
```
[metallb] Fix MetalLB migration issues and improve stability (#11224)

Signed-off-by: Denis Tarabrin <denis.tarabrin@flant.com>
Signed-off-by: Andrey Polovov <andrey.polovov@flant.com>
Co-authored-by: Andrey Polovov <andrey.polovov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [ea6acd](https://github.com/deckhouse/deckhouse/commit/ea6acdd008f788bd62f78f1f42d998990fda33d0) in branch **main**:
```
[cni-cilium] Fixing tunnelMode setting for Static clusters (#11233)

Signed-off-by: Andrey Polovov <andrey.polovov@flant.com>
Signed-off-by: Andrey Pavlov <andrey.pavlov@flant.com>
Co-authored-by: Andrey Pavlov <andrey.pavlov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [f467c6](https://github.com/deckhouse/deckhouse/commit/f467c6d3ea7d0c985b10686e0d0ef62d24ec29b2) in branch **main**:
```
chore: add dmt lint (#11112)

Signed-off-by: Kirill Kramorov <kirill.kramorov@flant.com>
Signed-off-by: Evsyukov Denis <denis.evsyukov@flant.com>
Co-authored-by: Kirill Kramorov <kirill.kramorov@flant.com>
Co-authored-by: Evsyukov Denis <denis.evsyukov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [5d7448](https://github.com/deckhouse/deckhouse/commit/5d744858db03c17112698b4fe9fc2d921fdcb9cf) in branch **main**:
```
[prometheus] Rename targets for alerts and docs fixes (#11243)

* [prometheus] Rename targets for alerts and docs fixes

Signed-off-by: alexey.komyakov <alexey.komyakov@flant.com>
---------

Signed-off-by: alexey.komyakov <alexey.komyakov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [6075dc](https://github.com/deckhouse/deckhouse/commit/6075dcbfe28767a01c4bd2ba1feee87dfd3be52d) in branch **main**:
```
[dhctl][deckhouse-controller] Fix vuln's (#11231)

Signed-off-by: Nikolay Mitrofanov <nikolay.mitrofanov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [be4006](https://github.com/deckhouse/deckhouse/commit/be4006853d60b64fd3a183998e2df42ed4579fe9) in branch **main**:
```
fix: add okmeter to exclude dmtlint (#11252)

Signed-off-by: Evsyukov Denis <denis.evsyukov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [be02c1](https://github.com/deckhouse/deckhouse/commit/be02c17f29e5f2a341af968359e52b51c74cf26e) in branch **main**:
```
[deckhouse] fix several deployed module releases (#11245)

Signed-off-by: Pavel Okhlopkov <pavel.okhlopkov@flant.com>
Co-authored-by: Pavel Okhlopkov <pavel.okhlopkov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [beb356](https://github.com/deckhouse/deckhouse/commit/beb356799e62a61121f703ae3ff0be5ed1aa732c) in branch **main**:
```
[descheduler] fix conversion (#11223)

Signed-off-by: Denis Romanenko <denis.romanenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [e9023a](https://github.com/deckhouse/deckhouse/commit/e9023a122bae38ae890531eabfff79826a99fb2c) in branch **main**:
```
[docs] Add detailed documentation for kubelet certificate rotation (#11139)


Signed-off-by: Artem Kulyabin <artem.kulyabin@flant.com>
Signed-off-by: Lada Lysenko <ladalada7878@gmail.com>
Signed-off-by: Artem Kladov <6360800+z9r5@users.noreply.github.com>
Co-authored-by: Artem Kulyabin <artem.kulyabin@flant.com>
Co-authored-by: Lada Lysenko <ladalada7878@gmail.com>
Co-authored-by: Artem Kladov <6360800+z9r5@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [0b6b02](https://github.com/deckhouse/deckhouse/commit/0b6b02799195fe560b4a0e5004b8514983fad790) in branch **main**:
```
[core] update base images to golang 1.23.4 (#11241)

Signed-off-by: Denis Romanenko <denis.romanenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [dcbc4c](https://github.com/deckhouse/deckhouse/commit/dcbc4c5f5e0d4128d49a3738d74bd58d0e64af7a) in branch **main**:
```
[monitoring] CNI misconfiguration alerts fix (#11259)

Signed-off-by: Andrey Polovov <andrey.polovov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [23cdce](https://github.com/deckhouse/deckhouse/commit/23cdcec5c872a2718b82cf0cc73993ff695007e9) in branch **main**:
```
[docs][virtualization] dvp admin install sync with GS (#10929)

Signed-off-by: Ivan Mikheykin <ivan.mikheykin@flant.com>
Signed-off-by: Zhbert <konstantin.nezhbert@flant.com>
Signed-off-by: mortelumina98 <ekaterina.matseevska@flant.com>
Signed-off-by: Ekaterina Matseevska <ekaterina.matseevska@flant.com>
Signed-off-by: Artem Kladov <artem.kladov@flant.com>
Co-authored-by: Zhbert <konstantin.nezhbert@flant.com>
Co-authored-by: mortelumina98 <ekaterina.matseevska@flant.com>
Co-authored-by: Artem Kladov <artem.kladov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [c0f91a](https://github.com/deckhouse/deckhouse/commit/c0f91a2b0a054bdf7cdb2b1203bf73c30a47e453) in branch **main**:
```
[docs][ingress-nginx] Update documentation (#11174)


Signed-off-by: DaNnNiS <galindanis5@gmail.com>
Signed-off-by: Max Chervov <max.chervov@gmail.com>
Signed-off-by: Artem Kladov <artem.kladov@flant.com>
Co-authored-by: Max Chervov <max.chervov@gmail.com>
Co-authored-by: Artem Kladov <artem.kladov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [43a500](https://github.com/deckhouse/deckhouse/commit/43a500cb4d783e4c009352e85ca25bc2d1f3cc3f) in branch **main**:
```
Re-generate changelog v1.67.2 (#11211)

Signed-off-by: deckhouse-BOaTswain <89150800+deckhouse-boatswain@users.noreply.github.com>
Co-authored-by: Taior <Taior@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [27110c](https://github.com/deckhouse/deckhouse/commit/27110c4e24d0820ec6b66edbcdf43c77ba5930a0) in branch **main**:
```
[containerd] fix runc tun device permission error (#11280)

Signed-off-by: Nikolay Gorbatov <nikolay.gorbatov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [2c220d](https://github.com/deckhouse/deckhouse/commit/2c220d150a63859f2d78235ea5868ca900ac2dcc) in branch **main**:
```
Re-generate changelog v1.67.2 (#11281)

Signed-off-by: deckhouse-BOaTswain <89150800+deckhouse-boatswain@users.noreply.github.com>
Co-authored-by: Taior <Taior@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [5fdada](https://github.com/deckhouse/deckhouse/commit/5fdada46adabaeda470cddb1075d39a591007c61) in branch **main**:
```
[user-authn] fix dexClient internal openapi spec (#11285)

Signed-off-by: Vasily Maryutenkov <vasily.maryutenkov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New tag **v1.67.3** is created.
